trivy-report/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Run Trivy vulnerability scanner
+description: Run Trivy vulnerability scanner and generate a report and save it to a Slack channel
+author: Extenda
+inputs:
+  image-ref:
+    description: The reference to the image to scan
+    required: true
+  application-name:
+    description: The name of the application
+    required: true
+  github-token:
```

Comment:
This should be `service-account-key` and it is not a github token. It's a GCP service account.